### ISSUE TITLE
Can't create cname in Vagrantfile

### ISSUE THREAD

**Issue by robb-reporo** [vagrant-landrush/landrush#176]
this works...

`system("vagrant landrush set #{hostalias} #{hostname})`

this does not work...

`config.landrush.host hostalias, hostname`

the following error is produced...

```
/opt/vagrant/embedded/lib/ruby/2.2.0/ipaddr.rb:559:in `in6_addr': invalid address (IPAddr::InvalidAddressError)
	from /opt/vagrant/embedded/lib/ruby/2.2.0/ipaddr.rb:496:in `initialize'
	from /home/vagrant/.vagrant.d/gems/gems/landrush-0.19.0/lib/landrush/action/setup.rb:66:in `new'
	from /home/vagrant/.vagrant.d/gems/gems/landrush-0.19.0/lib/landrush/action/setup.rb:66:in `block in setup_static_dns'
	from /home/vagrant/.vagrant.d/gems/gems/landrush-0.19.0/lib/landrush/action/setup.rb:61:in `each'
```

**Comment by njam** [comment]
What are the actual values that you are using? "hostalias" and "hostname"?

**Comment by robb-reporo** [comment]
This works...
`Landrush::Store.hosts.set(hostalias, hostname)`
so I am happy.

**Comment by robb-reporo** [comment]
> What are the actual values that you are using? "hostalias" and "hostname"?

hostalias is"foo" and hostname "foobox.vagrant", vagrant is set as our tld.

**Comment by robb-reporo** [comment]
The workaround given above does not work in 1.0.0.

**Comment by hferentschik** [comment]
Fix via pull request #207
